migration/00001_init_tables.sql

```diff
@@ -0,0 +1,31 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+
+CREATE TABLE "internals" (
```

Comment:
change to "clients"

Reply:
ok : 463ef200aaad80ec8fdf68f8ec38fcc4035b0c66